README.md

```diff
@@ -7,7 +7,7 @@
 
 > :warning: This project has not yet had a security audit or stress test and is therefore not ready for use in production! Use at your own risk!
 
-_Raftify_ is a Go implementation of the Raft leader election algorithm without the Raft log and enables the creation of a self-managing cluster of nodes by transforming an application into a Raft node. It is meant to be a **more cost-efficient** and **better-scaling** alternative to running a full-fledged Raft cluster with separate clients in an active/active setup.
+_Raftify_ is a Go implementation of the Raft leader election algorithm without the Raft log and enables the creation of a self-managing cluster of nodes by transforming an application into a Raft node. It is meant to be a **more cost-efficient** small-scale alternative to running an active/active validator cluster with a separate full-fledged Raft consensus layer.
```

Comment:
Not sure about this wording. I think we still count as active/active since every participating node has the means of signing a precommit.